deployment/ccip/changeset/v1_6/cs_add_new_chain_e2e.go

```diff
@@ -289,7 +289,9 @@ func addCandidatesForNewChainLogic(e deployment.Environment, c AddCandidatesForN
 	if err != nil {
 		return deployment.ChangesetOutput{}, fmt.Errorf("failed to load onchain state: %w", err)
 	}
-	donID, err := state.Chains[c.HomeChainSelector].CapabilityRegistry.GetNextDONId(&bind.CallOpts{
+
+	// get the nextDonID from donID claim to be claimed
+	donID, err := state.Chains[c.HomeChainSelector].DonIDClaimer.GetNextDONId(&bind.CallOpts{
```

Comment:
We will need to add some validation to this changeset that asserts that the DonIDClaimer exists on the home chain